textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/team3lucenequeryexample/LuceneQueryExample.java

```diff
@@ -0,0 +1,78 @@
+package edu.uci.ics.textdb.sandbox.team3lucenequeryexample;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.Tokenizer;
+import org.apache.lucene.analysis.ngram.NGramTokenizer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.FieldType;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexOptions;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+
+public class LuceneQueryExample {
+	public static void main(String[] args) throws Exception {
+		String dataFileName = "/Users/laishuying/Desktop/lucenetext.txt";
+		
+		Analyzer analyzer = new Analyzer() {
+			@Override
+			protected TokenStreamComponents createComponents(String fieldName) {
+				Tokenizer source = new NGramTokenizer(3,3);
+				return new TokenStreamComponents(source);
+			}
+		};
+		
+		Directory indexDir = FSDirectory.open(Paths.get("index"));
+		try {
+			IndexWriterConfig config = new IndexWriterConfig(analyzer);
+			IndexWriter writer = new IndexWriter(indexDir, config);
+			Scanner dataFile = new Scanner(new File(dataFileName));
+			Integer lineNum = 1;
+			while (dataFile.hasNextLine()) {
```

Comment:
@laisycs ... We can inject LuceneConfiguration (which we have to create) object to IDataReader/IDataWriter to customize the way we want to store the fields in Lucene Database. Since currently it is in sandbox, I suggest you to use indexWriter directly to experiment your changes. We can make the changes in IDataReader/IDataWriter to inject LuceneConfiguration, once you guys are ready to move your changes to the actual projects.